net/instaweb/rewriter/webscale_make_scripts_async.cc

```diff
@@ -0,0 +1,108 @@
+/*
+ *  COPYRIGHT:
+ *     (C) Copyright Webscale Networks 2017.
+ *      Licensed Materials - All Rights Reserved.
+ *    
+*/
+
+#include <vector>
+
+#include "base/logging.h"
+#include "net/instaweb/rewriter/public/rewrite_driver.h"
+#include "net/instaweb/rewriter/public/rewrite_driver_factory.h"
+#include "net/instaweb/rewriter/public/webscale_make_scripts_async.h"
+#include "pagespeed/kernel/html/html_element.h"
+#include "pagespeed/kernel/html/html_name.h"
+#include "pagespeed/kernel/html/html_node.h"
+#include "pagespeed/kernel/base/message_handler.h"
+#include "pagespeed/kernel/base/statistics.h"
+#include "pagespeed/kernel/base/string.h"
+#include "pagespeed/kernel/base/string_util.h"
+#include "pagespeed/kernel/util/re2.h"
+
+namespace net_instaweb {
+
+WebscaleMakeScriptsAsync::WebscaleMakeScriptsAsync(RewriteDriver* rewrite_driver, MessageHandler* message_handler)
+    : CommonFilter(rewrite_driver) {
+  Statistics* statistics = rewrite_driver->statistics();
+  message_handler = driver()->message_handler();
```

Comment:
you are assigning the parameter here, not the member variable, right?  Because the parameter is shadowing `this->message_handler`.  Use `this->`